### PR TITLE
use custom template for Cost resource step

### PR DESCRIPTION
**Pull request by PanSpagetka**
it sets max-height for selects on Cost resource step. Ale styles for this step wasn't applied, so I fixed it by adding custom template. 

https://issues.redhat.com/browse/RHCLOUD-13983

### CODE REVIEW

File: src/smart-components/role/add-role-new/cost-resources-template.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Title } from '@patternfly/react-core';
+
+const CostResourcesTemplate = ({ formFields }) => {
+  console.log('PRDEL', formFields);
```
Comment (Hyperkid123) [inline src/smart-components/role/add-role-new/cost-resources-template.js:6]: :eye:

Comment (PanSpagetka) [reply src/smart-components/role/add-role-new/cost-resources-template.js:6]: ups

File: src/smart-components/role/add-role-new/cost-resources.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { Select, SelectOption, SelectVariant, Grid, GridItem, Text, TextVariants
 import useFieldApi from '@data-driven-forms/react-form-renderer/dist/esm/use-field-api';
 import useFormApi from '@data-driven-forms/react-form-renderer/dist/esm/use-form-api';
 import { getResourceDefinitions, getResource } from '../../../redux/actions/cost-management-actions';
+import './add-role-wizard.scss';
```
Comment (Hyperkid123) [inline src/smart-components/role/add-role-new/cost-resources.js:7]: Do we need this import? I would assume it was imported before. We should avoid duplicate imports.

File: src/smart-components/role/add-role-new/cost-resources-template.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Title } from '@patternfly/react-core';
+
+const CostResourcesTemplate = ({ formFields }) => (
+  <div className="rbac">
+    <Title headingLevel="h1" size="xl" className="pf-u-mb-lg">
+      Review details
+    </Title>
+    {[...formFields]}
```
Comment (Hyperkid123) [inline src/smart-components/role/add-role-new/cost-resources-template.js:10]: Is it necessary to duplicate the array from props?

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**